site/management.md

```diff
@@ -335,6 +348,61 @@ curl -i --header "authorization: Bearer &lt;token&gt;" http://localhost:15672/ap
 This is true for all endpoints except `GET /definitions` and `POST /definitions`. Those
 endpoints require the token to be passed in the `token` query string parameter.
 
+### Configure Scopes to request during authentication
+
+We can configure which OAuth2 **scopes** RabbitMQ should claim when redirecting the user to the authorization server by
+setting `management.oauth_scopes`.
+
+If we do not specify any scopes, when `management.enable_uaa = true`, RabbitMQ defaults to
+<pre class="lang-ini">
+management.oauth_scopes = "openid profile rabbitmq.*".
+</pre>
+
+Where `rabbitmq` is the *resource_server_id* we declared in the oauth2 plugin configuration:
+<pre class="lang-ini">
+ {rabbitmq_auth_backend_oauth2, [
+    {resource_server_id, &lt;&lt;"rabbitmq"&gt;&gt;},
+</pre>
+
+In the contrary, when `management.enable_uaa = false` or when it is not set, RabbitMQ defaults to
```

Comment:
I would remove unnecessary words in documentation - I don't think we need "In the contrary" please remove